app/classes/person.py

```diff
@@ -0,0 +1,318 @@
+            print 'There are no rooms in the system'
+        else:
+            # Populating the room key
+
+            for room_key, room in rooms.items():
+                room_id.append(room_key.upper())
+
+            # Checking if room is in the room key
+
+            if work_space.upper() in room_id:
+
+                # Createing a holder dictionary for display
+
+                r_key = {work_space.upper(): ''}
+
+                for person_key, person in persons_data.items():
+
+                    person_room = person['work_space'].upper()
+
+                    if person_room == work_space.upper():
+                        r_key[person_room] += (" ".join(person['name']) + "\n")
+
+                for room_info in r_key:
+                    print room_info.upper()
+                    print '_' * 20
+                    print r_key[room_info]
+                    print '_' * 20
+
+            else:
+                print 'There is no room named ' + work_space.upper()
+
+    def print_unallocated(self):
+
+        """
+        Prints Unallocated people
+
+        """
+        unallocated_list = []
+
+        for key, person in Person.people_list.items():
+
+            if person['work_space'] == 'None':
+                unallocated_list.append(person['name'])
+
+        if len(unallocated_list) == 0:
+            return 'There are no unallocated people'
+        else:
+            return unallocated_list
+
+    def load_people(self, filename):
+
+        """
+        Adding people from a text file
+
+        """
+        with open(filename, 'r') as people_file:
+            people = people_file.readlines()
+
+            for person in people:
+                person = person.split(' ')
+
+                if len(person) == 3:
+                    # if there are three values
+                    first_name = person[0]
+                    second_name = person[1]
+                    job_type = person[2].strip()
+                    wants_accommodation = "N"
+                    employee_number = randint(0,1000)
+                    self.add_person(first_name,second_name, job_type.upper(),employee_number,
+                                    wants_accommodation)
+
+                elif len(person) == 4:
+                    # If there are four values
+                    first_name = person[0]
+                    second_name = person[1]
+                    job_type = person[2].strip()
+                    employee_number = randint(0,1000)
+                    wants_accommodation = person[3].strip()
+
+                    self.add_person(first_name, second_name, job_type.upper(), employee_number,
+                                    wants_accommodation)
+
+
+                else:
+                    return 'Not available'
+
+            return 'successfully added'
+
+
+
+
+class Fellow(Person):
+
+    """docstring for FELLOW, subclass of Person."""
+
+    def __init__(self, arg):
+        super(fellow, self).__init__()
```

Comment:
Double check on naming in this function, it should be `super(Fellow, self).__init__()`